### DISCUSSION TITLE
AWS deploy?

### DISCUSSION THREAD

**Discussion by coman-rpx** [blitz-js/blitz#1436] [Q&A]
Was wondering if anyone has deployed a blitz project to AWS?   I noticed that prisma has an AWS deploy example: https://www.prisma.io/docs/guides/deployment/deploying-to-aws-lambda, which I tried and worked beautifully, but I'm a little unclear how to do something similar with blitz....thanks for any suggestions.

**Comment by jamiedavenport** [comment]
There's likely lots of different ways to achieve deploying Blitz to AWS (EC2, ECS etc) and different ways of managing the infrastructure (Terraform, CloudFormation etc). I'm curious to see what setups are most popular within the community.

Personally I'd love docs / a recipe for deploying with Beanstalk and I think that aligns with the goals of Blitz.

**Reply by jamiedavenport** [reply]
https://github.com/blitz-js/blitzjs.com/issues/247

**Comment by flybayer** [comment]
Yeah, I know several people have. I'll post this in slack to see if any can give you pointers.

What specifically are you wandering about? Lambda? Or like elastic beanstalk?

**Reply by coman-rpx** [reply]
Thanks for all the replies, 
I'm just trying to figure out a super simple workflow using EB + a RDS postgres db running Blitz.
